cmds/alias.c

```diff
@@ -30,38 +35,79 @@ static int cmd_alias(int argc, char *argv[])
 	char *end;
 	size_t sz = 0;
 	addr_t addr = 0;
-	unsigned int i;
+	int c, relative = 0;
+	addr_t newBase;
 
 	if (argc == 1) {
+		lib_printf("\nRelative alias base: 0x%p", aliasBase);
 		phfs_aliasesShow();
-		return EOK;
+		return CMD_EXIT_SUCCESS;
+	}
+
+	for (;;) {
+		c = lib_getopt(argc, argv, "rb:");
+		if (c < 0) {
+			break;
+		}
+
+		switch (c) {
+			case 'r':
+				relative = 1;
+				break;
+
+			case 'b':
```

Comment:
`argc` could be checked. I think it should be, as it's not obvious that `alias -b 0x10000000 test 0x200 0x200` (which returns no error) doesn't create alias

Reply:
Done